src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1004,7 +1004,6 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
             // If this is a DesignSpecification:OutdoorAir:SpaceList check to make sure spaces are valid and belong to this zone
             if (thisOAReq.numDSOA > 0) {
                 for (int spaceCounter = 1; spaceCounter <= thisOAReq.numDSOA; ++spaceCounter) {
-                    std::string thisSpaceName = thisOAReq.dsoaSpaceNames(spaceCounter);
```

Comment:
[src/EnergyPlus/ZoneEquipmentManager.cc:1007]:(style),[unreadVariable],Variable 'thisSpaceName' is assigned a value that is never used.